helm/hpcc/templates/esp.yaml

```diff
@@ -75,6 +75,9 @@ data:
       dfuQueues:
 {{- include "hpcc.generateConfigDfuQueues" . | indent 6}}
 {{ include "hpcc.generateVaultConfig" . | indent 6 }}
+{{ if eq .me.application "eclwatch" }}
```

Comment:
minor: this will 'leak' a newline, should I think be (but should be checked):
```
{{- if eq .me.application "eclwatch" }}
```
similarly with the end

helm template whitespace chomping is horrid!

Reply:
I removed it because the formatting became messed up - it was reporting that it couldn't convert to json.  But I;ve just adding it back in and I can't see the original problem anymore.

Reply:
as I say, getting whitespace chomping right is horrible. You have to be really careful, not to remove a newline where you need it etc.